doc/configuration.txt

```diff
@@ -2958,6 +2959,9 @@ maxzlibmem <number>
   with "show info" on the line "MaxZlibMemUsage", the memory used by zlib is
   "ZlibMemUsage" in bytes.
 
+mptcp
+  uses MPTCP instead of TCP.
```

Comment:
Capital letter: 'Uses'

While at it:

```
  Uses MPTCP instead of TCP. Multipath TCP or MPTCP is an extension to the
  standard TCP and is described in RFC 8684. It allows a device to make use of
  multiple interfaces at once to send and receive TCP packets over a single
  MPTCP connection. MPTCP can aggregate the bandwidth of multiple interfaces or
  prefer the one with the lowest latency, it also allows a fail-over if one path
  is down, and the traffic is seamlessly reinjected on other paths.
```

(from mptcp.dev)